modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -98,7 +98,7 @@ function MultipleChoice(props) {
   const [ghostName, setGhostName] = useState(inputPrefill?.displayedValue);
   const [ghostValue, setGhostValue] = useState(inputPrefill?.value || GHOST_SENTINEL);
   const ghostSelected = selection.some(element => {return element.includes(ghostValue);});
-  const disabled = maxAnswers > 0 && selection.length >= maxAnswers && !isRadio && !ghostSelected;
+  const disabled = maxAnswers > 0 && selection.length >= maxAnswers && maxAnswers !== 1 && !ghostSelected;
```

Comment:
I think this can be simplified as:
```suggestion
  const disabled = maxAnswers > 1 && selection.length >= maxAnswers;
```